lnwallet/channel.go

```diff
@@ -5981,9 +5981,19 @@ func (lc *LightningChannel) availableBalance() (lnwire.MilliSatoshi, int64) {
 	// add updates concurrently, causing our balance to go down if we're
 	// the initiator, but this is a problem on the protocol level.
 	ourLocalCommitBalance, commitWeight := lc.availableCommitmentBalance(
-		htlcView,
+		htlcView, false,
 	)
 
+	// Do the same calculation from the remote commitment point of view.
+	ourRemoteCommitBalance, _ := lc.availableCommitmentBalance(
```

Comment:
It looks like `availableBalance` does many of the things that `validateCommitmentSanity` does too. Is it possible to unify both calls by just having `validateCommitmentSanity` return the amount sendable too? That way we can also be more sure that `availableBalance` really returns the maximum that `AddHtlc` would accept.

Reply:
Added TODO.